vendor/github.com/coreos/etcd/cmd/tools

```diff
@@ -0,0 +1 @@
+../tools
```

Comment:
Why are there more symbolic link files than before?

Reply:
I have no idea. Maybe people from etcd can answer this.

Reply:
@zhexuany can we use the clean_vendor script to remove all the files under `cmd` folder?

Reply:
I think that would be too specific. It may cause some trouble when the dependent project changes directory structure. /cc @coocood